src/error.rs

```diff
@@ -13,31 +13,37 @@ use core::fmt;
 use std::{io, error};
 
 // A randomly-chosen 16-bit prefix for our codes
-pub(crate) const CODE_PREFIX: u32 = 0x57f40000;
+pub(crate) const CODE_PREFIX: u32 = 0x57f4c500;
```

Comment:
Apparently this is now a 24-bit prefix.

I was thinking of allowing an 8-bit section per platform for platform-specific constants, hence leaving 16-bits zero, but I guess it's not too important.

Reply:
Hm, platform specific codes are defined by platform and not by us, so I am not sure how it would work.

Reply:
Not existing platform-specific codes, but at least the wasm-stdweb impl seems to want its own codes.

Reply:
Hm, how about simply listing those platform-specific error codes in documentation?